recipe-server/client/control_new/components/recipes/RecipeListing.js

```diff
@@ -74,7 +75,18 @@ export default class RecipeListing extends React.Component {
           title="Name"
           dataIndex="name"
           key="name"
-          render={RecipeListing.renderLinkedText}
+          render={(text, record) =>
+            (
+              <div className="recipe-listing-name">
+                <ShieldIdenticon
+                  className="shieldicon"
+                  seed={record.id}
+                  size={32}
+                />
+                {RecipeListing.renderLinkedText(text, record)}
```

Comment:
I don't see any other instances of `renderLinkedText` being called. Perhaps instead of adding the `div.recipe-listing-name` here it can be moved to the `renderLinkedText` function to keep this JSX simple (`render={RecipeListing.renderLinkedText}`) (Note that a different name for the function may be more suitable too).

If `renderLinkedText` is used elsewhere though, disregard this note entirely.

Reply:
`renderLinkedText` is also used for the Action column.